src/literal.h

```diff
@@ -665,6 +661,9 @@ class Literal {
   Literal relaxedFmaF64x2(const Literal& left, const Literal& right) const;
   Literal relaxedFmsF64x2(const Literal& left, const Literal& right) const;
 
+  Literal externalize() const;
```

Comment:
Please document in this header how the encoding is done for externalized/internalized data.